btree/btree.c

```diff
@@ -3693,11 +3705,10 @@ static bool child_node_check(struct m0_btree_oimpl *oi)
 
 /**
  * This function will determine if there is requirement of loading root child.
- * If root contains only two record and if any of them is going to get deleted
- * we might want to load the other child of root as well to handle root
- * underflow.
+ * If root contains only two records and if any of them is going to get deleted,
+ * it is required to load the other child of root as well to handle root case.
  *
- * @param bop It will provide all required information about btree operation.
+ * @param bop it will provide all required information about btree operation.
```

Comment:
You need to remove 'it' after bop. Check the other parameters how they are described.